frontend/src/app/history-query/resetCache-history-query-modal/resetCache-history-query-modal.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'oib-resetCache-history-query-modal',
+  templateUrl: './resetCache-history-query-modal.component.html',
+  styleUrl: './resetCache-history-query-modal.component.scss',
```

Comment:
Please do not use camelCase inside angular file name. Rename them reset-cache-history-query-modal instead